packages/core/components/FileDetails/index.tsx

```diff
@@ -75,20 +76,53 @@ export default function FileDetails(props: Props) {
     const [thumbnailPath, setThumbnailPath] = React.useState<string | undefined>();
     const [isThumbnailLoading, setIsThumbnailLoading] = React.useState(true);
     const stackTokens: IStackTokens = { childrenGap: 12 + " " + 20 };
+    const [calculatedSize, setCalculatedSize] = React.useState<number | null>(null);
+
+    const platformDependentServices = useSelector(
+        interaction.selectors.getPlatformDependentServices
+    );
+    const fileDownloadService = platformDependentServices.fileDownloadService;
+    const isOnWeb = useSelector(interaction.selectors.isOnWeb);
+    const isZarr = fileDetails?.path.endsWith(".zarr") || fileDetails?.path.endsWith(".zarr/");
```

Comment:
Oh I hadn't realized ".zarr/" would be a possibility. I bet I have to refactor some regex to account for this